h/static/scripts/drafts.js

```diff
@@ -19,6 +19,14 @@ function DraftStore() {
   }
 
   /**
+   * Returns a list of all drafts - both unsaved new annotations, and unsaved
+   * edits to saved annotations.
+   */
+  this.drafts = function drafts() {
+    return this._drafts.slice();
```

Comment:
AFAICT you only ever use this to count the drafts. Perhaps just make a `count` function that returns `this._drafts.length` instead of copying the entire array?